templates/database_docker_backup.sh.j2

```diff
@@ -62,7 +77,17 @@ do
   BACKUP_FOLDER=$CURRENT_DIR/backups/$i
   mkdir -p $BACKUP_FOLDER
   docker exec $i sh -c 'mongodump -u "$MONGO_INITDB_ROOT_USERNAME" -p "$MONGO_INITDB_ROOT_PASSWORD" --archive' | zstd > $BACKUP_FOLDER/$(date +%Y%m%d-%H%M).archive.zst
+{% if docker_database_backup_ping | length %}
+  SUM_OF_EXIT_CODES+=${PIPESTATUS[0]}
+  SUM_OF_BACKUPS+=1
+{% endif %}
 done
 
 # delete backups older than 14 days
-/usr/bin/find $CURRENT_DIR/backups/ -mtime +$KEEP_DAYS -delete -type f
+/usr/bin/find $CURRENT_DIR/backups/ -mtime +$KEEP_DAYS -type f -delete
+
+{% if docker_database_backup_ping | length %}
+# ping monitoring
+ping_body="[${SUM_OF_BACKUPS}x] $(tail -n 4 {{ docker_database_backup_path }}/last_run.txt)"
```

Comment:
you can use `$CURRENT_DIR`, which is used generally in the script, instead of `{{ docker_database_backup_path }}`